sass/_theme_layout.sass

```diff
@@ -2,6 +2,15 @@
 html
   font-size: $base-font-size
 
+// Set uniform form font styles
+// This prevents Firefox on Linux from rendering forms with dark styling
+// when using a dark system theme, which looks bad on an otherwise light
+// website theme
+input
```

Comment:
The selector for other search field styling is `.wy-side-nav-search input[type=text]`. Would be better to add the styling there to prevent accidentally restyling other input elements. The theme may not use them, but Sphinx extensions may include them and we don't want to apply our styles there accidentally.

Reply:
I'd like the style to affect this form as well:

![image](https://user-images.githubusercontent.com/180032/60683959-39e5ed00-9e9b-11e9-8475-58d5ed43e643.png)

If I add to the `.wy-side-nav-search input[type=text]` selector, it won't affect that form anymore.

Reply:
OK, then a better selector would be `input[type=text]`